plugins/org.jboss.tools.openshift.ui/META-INF/MANIFEST.MF

```diff
@@ -28,6 +28,10 @@ Export-Package: org.jboss.tools.openshift.internal.ui;x-friends:="org.jboss.tool
  org.jboss.tools.openshift.internal.ui.command;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.deployment;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.explorer;x-friends:="org.jboss.tools.openshift.test",
+ org.jboss.tools.openshift.internal.ui.job,
  org.jboss.tools.openshift.internal.ui.jobs;x-friends:="org.jboss.tools.openshift.test",
  org.jboss.tools.openshift.internal.ui.property;x-friends:="org.jboss.tools.openshift.test",
- org.jboss.tools.openshift.internal.ui.property.build;x-friends:="org.jboss.tools.openshift.test"
+ org.jboss.tools.openshift.internal.ui.property.build;x-friends:="org.jboss.tools.openshift.test",
+ org.jboss.tools.openshift.internal.ui.wizard.application
+Import-Package: org.apache.commons.io,
+ org.eclipse.jface.text
```

Comment:
please replace a specific package import of Eclipse classes by a required bundle that provides this package (we should not mix approaches: we should either consistently only import packages or require bundles)